### DISCUSSION TITLE
Messages: BR-BT-00140-0004 vs BR-BT-00140-0054

### DISCUSSION THREAD

**Discussion by AnnaKSZ** [OP-TED/eForms-SDK#475] [Q&A] [answered]
Hi Lab!

I have found these messages: 

BR-BT-00140-0004
BT-140-notice is mandatory for a notice with subtype 1 under the condition: Change Notice Version Identifier (BT-758-notice) is **not** present

BR-BT-00140-0054
BT-140-notice is forbidden for a notice with subtype 1 under the condition: Change Notice Version Identifier (BT-758-notice) is **not** present

The same conditions make me the question. Is this correct?
I did not find this message code in the SDK other than rules translation files.

BR-BT-00140-0004 -> I think here must be present the BT-758, or am I wrong?
(from BR-BT-00140-0001 to BR-BT-00140-0052.)

Regards
Anna

**Comment by YvesJo** [comment, marked as answer]
Your observation is correct and the statement for rule BR-BT-00140-0004 should read:

> _BT-140-notice is mandatory for a notice with subtype 1 under the condition: Change Notice Version Identifier (BT-758-notice) is present_

This rule is not yet active. Labels for Conditional rules were generated in bulk and the labels for conditionally mandatory rules are sometimes improperly stated. These statements are reviewed and updated as the rules get activated.
